+hw/Timeline.m

```diff
@@ -604,7 +604,8 @@ function livePlot(obj, data)
             %   TL.LIVEPLOT(source, event) plots the data aquired by the
             %   DAQ while the PlotLive property is true.
             if isempty(obj.Axes)
-                f = figure('Units', 'Normalized', 'Position', [0 0 1 1]); % create a figure for plotting aquired data
```

Comment:
I'd leave this in as all timeline computers run solely for that purpose and usually there are many inputs being acquired so more often than not the experimenter would want it to be full screen

Reply:
but this allows for optionality? can still make figure full screen